y_lb.info.yml

```diff
@@ -16,4 +16,7 @@ dependencies:
   - drupal:lb_claro
   - drupal:scheduler
   - drupal:pathauto
+  - drupal:field_group
+  - drupal:metatag
+  - drupal:openy_focal_point
```

Comment:
there is no such module at drupal.org https://www.drupal.org/project/openy_focal_point

```suggestion
  - openy_focal_point
```

Reply:
See https://ds-docs.y.org/docs/wiki/dependencies-in-drupal-info.yml/ @froboy

Reply:
Also, see https://github.com/YCloudYUSA/y_lb/pull/122/commits/d6363b975dfdae6ab6a1696cdcbb2f26ddab951a @froboy